docs/docs/modules.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: default
+title: KAMI Blue Modules
+description: KAMI Blue Module List
+---
+
+## KAMI Blue Modules
```

Comment:
This is missing a back button like the other pages

Reply:
LOL. I assumed the title "KAMI Blue" in the banner was a link back to home like literally every website ever. Seems like a better solution than having a `<back>` link on every page but whatever

Reply:
No, if you looked at any of the other pages you would see the title is actually a title...

Reply:
I can add a link ref but it's unintuitive.

Reply:
> No, if you looked at any of the other pages you would see the title is actually a title...

Eh, that's a pretty bold assumption. People don't always stylistically distinguish links, especially when they're also a title

Reply:
They aren't links? I'm saying the titles on my pages are actually titles

Reply:
I can add a default header to go back, but having a back button is more obvious.